README.md

```diff
@@ -126,36 +126,37 @@ If you find issues with this application suite please create an [issue](https://
 Contributions are welcome. Please ensure they relate to an issue. See our 
 [Code of Conduct](./CODE-OF-CONDUCT.md) that is included with this repo for important details on contributing to Government of British Columbia projects. 
 #### How to setup Local development
-We suggest run this in docker container, you can download docker [here](https://www.docker.com/get-started).
+We suggest running this in a docker container, you can download docker [here](https://www.docker.com/get-started).
 
-This application has a function that will invite GitHub users to specified git organization, to make it locally, you will need to create your own test GitHub organization and Git App.
+This application has a function that will invite GitHub users to a specified GitHub organization, to make it locally, you will need to create your own test GitHub organization and Git App.
 
 **Step 1**
-Create a/two Git Organization(s) [here](https://github.com/account/organizations/new?coupon=&plan=team_free) or you can use any exiting orgnization that you have full access.
+Create a/two GitHub Organization(s) [here](https://github.com/account/organizations/new?coupon=&plan=team_free) or you can use any existing organization that you have full access permissions organization
+(ps*: API can invite the user into one or multiple GitHub org(s). for testing and development purpose, we suggest you at least create two GitHub orgs to make sure this function works properly)
 
 **Step 2**
-* In organization setting => Developer settings => GitHub Apps create a new GitHub app.
+* Under organization setting => Developer settings => GitHub Apps create a new GitHub app. (`https://github.com/organizations/${REPLEASE WITH_YOUR_ORG_NAME}/settings/apps`)
```

Comment:
> REPLEASE

REPLACE? Also, unless they have set a variable common syntax would be: `[REPLACE_WITH_YOUR_ORG_NAME]` instead of the `${}` which is typically an inline variable of some sort